script/upload.py

```diff
@@ -93,8 +93,10 @@ def yield_brave_packages(dir, channel, version):
   for _, _, files in os.walk(dir):
     for file in files:
       if PLATFORM == 'darwin':
+        sought_file = 'Brave-Browser-' + channel.capitalize() + '.dmg'
         if re.match(r'Brave Browser ' + channel.capitalize() + r'.*\.dmg$', file):
-          yield file
+          os.rename(file, sought_file)
+          yield sought_file
```

Comment:
elif re.match(r'Brave-Browser-' + channel.capitalize() + r'.*\.dmg$', file):
    yield sought_file

Reply:
this is needed in case file upload is re-attempted and the file has been previously been renamed